tests/ztimer_periodic/main.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2020 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2020 Freie Universität Berlin
+ *               2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     test
+ * @{
+ *
+ * @file
+ * @brief       ztimer periodic test application
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <inttypes.h>
+
+#include "ztimer.h"
+#include "ztimer/periodic.h"
+#include "fmt.h"
+#include "mutex.h"
+
+static mutex_t _mutex = MUTEX_INIT_LOCKED;
+
+#define MAX_OFFSET 2
+#define N 5
+#define INTERVAL 100LU
+
+static volatile uint32_t _times[N];
+
+static int callback(void *arg)
+{
+    (void)arg;
+    static int count = 0;
+
+    _times[count] = ztimer_now(ZTIMER_MSEC);
+
+    count += 1;
+
+    if (count == N) {
+        mutex_unlock(&_mutex);
+    }
+
+    return (count == N);
+}
+
+int main(void)
+{
+    ztimer_periodic_t t;
+
+    ztimer_periodic_init(ZTIMER_MSEC, &t, callback, "ping", INTERVAL);
+    uint32_t last = ztimer_now(ZTIMER_MSEC);
+
+    ztimer_periodic_start(&t);
+
+    // wait for periodic to trigger N times
```

Comment:
Don't we usually use  doxygen syle comments?

Reply:
yes, totally right. doing too much rust lately... will fix